lib_eio_linux/eio_linux.ml

```diff
@@ -821,6 +821,10 @@ module Objects = struct
       end;
       if reuse_addr then
         Unix.setsockopt sock_unix Unix.SO_REUSEADDR true;
+
+      if reuse_port then
+        Unix.setsockopt sock_unix Unix.SO_REUSEPORT true;
+
       let sock = FD.of_unix ~sw ~seekable:false sock_unix in
```

Comment:
```suggestion
      if reuse_port then
        Unix.setsockopt sock_unix Unix.SO_REUSEPORT true;
      let sock = FD.of_unix ~sw ~seekable:false sock_unix in
```

Minor style nit: removed the blank lines for consistency with the rest of the eio code. You can use a comment if you want to make a block of code stand out, but I don't think we want that here.